lib/improver/tests/ensemble_calibration/ensemble_calibration/helper_functions.py

```diff
@@ -44,16 +43,18 @@
 from improver.utilities.cube_manipulation import concatenate_cubes
 
 
-def set_up_probability_above_threshold_cube(
+def set_up_probability_threshold_cube(
         data, phenomenon_standard_name, phenomenon_units,
         forecast_thresholds=np.array([8, 10, 12]), timesteps=1,
-        y_dimension_length=3, x_dimension_length=3):
+        y_dimension_length=3, x_dimension_length=3,
+        relative_to_threshold='above'):
```

Comment:
The changes to generalise this aren't necessary - there is work going on in #742 to replace these set up functions.  Adding an optional argument `relative_to_threshold` is OK but please would you revert the function name, as this creates a lot of extra lines of code to review for no reason.

Reply:
I don't quite follow. The reason the name has been changed is that this function no longer creates a probability cube above a threshold, it created a cube that is relative to the threshold as defined by the relative_to_threshold argument. If I change the name back we will have a function with a name that does not describe what it is doing.

The addition of this keyword was made to remove a copy/paste of this complete function that had been introduced to create cubes below a threshold.

Reply:
OK.